steampy/client.py

```diff
@@ -269,6 +266,11 @@ def make_offer_url(self, items_from_me: List[Asset], items_from_them: List[Asset
         token_start=trade_offer_url.index('token=')+len('token=')
         token=trade_offer_url[token_start:]
 
+        partner_account_id_start = trade_offer_url.index('?partner=')+len('?partner=')
+        partner_account_id_end = trade_offer_url.index('&token=')
+        partner_account_id = trade_offer_url[partner_account_id_start:partner_account_id_end]
+        partner_steam_id=account_id_to_steam_id(partner_account_id)
+
```

Comment:
if partner_account_id is not used anywhere extract lines 269-273 to another method.

Generally speaking when you make blank lines in code (like 268 and 273) then you highlighting some parts of code, and it is a good idea to extract it to method for clean code :)